### ISSUE TITLE
Joystream.org Homepage Hi-fi (Stage A)

### ISSUE THREAD

**Issue by dmtrjsg** [Joystream/atlas#2960]
## Context
* https://github.com/Joystream/marketing/issues/259

:info: This issue will get migrated to Marketing repo. Raising in Atlas while waiting for access rights and Zenhub integration with marketing

## Scope
Preview:
![Screenshot 2022-07-04 at 13.50.36.png](https://images.zenhubusercontent.com/6197874e1913711bf1f6ad31/69e26a3e-4811-4da0-bb8b-36caa0f6ceb7)

* [ ] Create lo-fi mockups to accommodate the above.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201958687417145/1202548025557190) by [Unito](https://www.unito.io)

**Comment by toiletgranny** [comment]
Alright, @bedeho, @dmtrjsg the HiFi designs for the new Joystream.org homepage are ready to be reviewed! Find out all about it from:

- [This Loom video update](https://www.loom.com/share/9657602415164e3e87ffbb3e9c2f1e12) (24 min at 1x speed, 16 min at 1.5x speed)
- [This Figma prototype](https://www.figma.com/proto/3uoobZkRhFF4yibFjCWa2D/Joystream-%7C-Website?page-id=5523%3A129021&node-id=5682%3A158884&scaling=min-zoom&starting-point-node-id=5682%3A158884). Please note the `A/B/C` variant controls occasionally included on the left-hand side of the prototype.

**Comment by dmtrjsg** [comment]
@toiletgranny amazing work 🤩  

My 5cents:

1. Loving the dynamic type. “Built on blockchain..” Shall we say “Blockchain Dapp managed by Joystream DAO”?
2. Option number 1 is winning imho, b4. Payouts for the scheme would be possible ahead of it although a reduced version of it. On that basis I suggest to remove “Coming this fall” label. 
3. “Oh and by the way section”  - Amazing!! Like the version 1 visuals and v2 copy more (less “today” twice)
4. Visuals on “Monetise your videos” with NFTs -> second version is better, cleaner and more benefits straight away.
5. Recently sold NFTs need to be in JOY (as Fiat exchange rate depends on listing and this may take some time)
6. CRTs -> Sign up for beta tests! Amazing. 
7. CRTs visuals -> second version is better but I feel another iteration may need to be shown including holders view. 
8. Second CTA claiming channel -> Brighter one is good! Fomo is better imho.
9. Manifesto breaks the flow if too early, I’m in your camp @Adam + manifesto as top nav works for me
10. Ativities: like the change of copy
11. Meet the team -> imho we don’t need it that bright. We don’t need to invest much into the banner, so just the three execs are good :)  I’d even say remove this section only leave link in the footer.
12. Effects: parallax, glare - yes. Dynamic headers - not sure, imho better legibility wo them :)

**Comment by bedeho** [comment]
First off, just phenomenal work, video and prototype, the variation switches were extremely helpful.

- I like the dynamic title, works well.
- NFT variation (A) is best for main product shot.
- There is no problem using a YT logo in thi context, I think it actually should be emphasised much more strongly. You once made a really nice graphic for this for a slide deck I used, and everyone understood it right away, attaching below.
- I think the NFT tile component (B) worked better, because it was a more focused and simple motif, the other one had much more going on, and was thus harder to process.
- I think the bright color sections work well, but I do feel the high contrast marks, with thick slopes and lots of motion, do take away from the text, which is the main content. 
- Did we decide to skip creators inducted? Just like "recently sold videos", it would be great ot be able to feature recently inducted creators from YT, with their 
- Could the payout section be strengthened by adding a carousel of actualy "rcently paid out" creators, with what they earned?
- I think the creator token visuals (A) similar to the NFT visuals worked best.
- I agree that the quote was more impactful.
- Lets indeed keep manifesto section towards bottom.
- The improved copy helped the activitie section
- I think we should keep the tema section simple, certainly anything dynamic is not worth it.
- Parallax & highlight would be really nice!

![Screenshot 2022-07-19 at 23 54 58](https://user-images.githubusercontent.com/437292/179855256-c74358d0-fbe9-4668-8d04-77b3504e63f2.png)

**Comment by mrexpat** [comment]
Beautiful!! @toiletgranny

**Comment by toiletgranny** [comment]
@dmtrjsg 
1.  > Loving the dynamic type. “Built on blockchain..” Shall we say “Blockchain Dapp managed by Joystream DAO”?

I like the simplicity, but is it an app, though? It's more of a platform, a protocol, isn't it. How about we just get rid of the "and" and say "Built on blockchain. Managed by DAO"?

**Comment by dmtrjsg** [comment]
Action points:
1. CRTs section - additional iteration agreed rejigging visuals and focus on benefits (raise money, community becomes stakeholder, revenue share scheme) 
2. Add recently paid out channels (as part of YPP programme) similar to recently sold NFTs carousel
3. Recently sold NFTs - design two versions of carousel - JOYs only and with USD denomination
4. Quote - split from CTA claim channel, try blending in between NFTs and CRTs. Fomo version is agreed to be taken ahead for claim channel CTA reinforcement.

🆕 (just thought ab it) Together with activities in the header or right after as separate banner, imho worth mentioning that One can partake in DAO activities and view proposals and forum via separate FE app - Pioneer and link to it.

cc @toiletgranny

**Comment by bedeho** [comment]
> One can partake in DAO activities and view proposals and forum via separate FE app - Pioneer and link to it.

Agreed, one could go one step further can add a carousel which shows recent proposals for example also, making it really come to life, just a thought!

**Comment by dmtrjsg** [comment]
@toiletgranny 

Info for proposal cards:


Title
Proposer (avatar of member)
Status (Deciding, Gracing or Executed)
Time left (for deciding and gracing); Date Executed (For executed)
Current Votes (only show the ones that are non-zero)
<img width="302" alt="Screenshot 2022-07-21 at 15 46 07" src="https://user-images.githubusercontent.com/93924525/180243415-0e3e7b6c-9106-434f-85f7-4d762dce836f.png">

 CTA > View proposal discussion

**Comment by toiletgranny** [comment]
Alright, let's see what we got! @bedeho, @dmtrjsg Here's all about the new iteration:

- [Loom video update](https://www.loom.com/share/0018f54e873c46a9bd81603e9fc16ce5) (14 min at 1x speed, 10 min at 1.5x speed)
- [Figma prototype](https://www.figma.com/proto/3uoobZkRhFF4yibFjCWa2D/Joystream-%7C-Website?page-id=5523%3A129021&node-id=5754%3A245848&scaling=min-zoom&starting-point-node-id=5754%3A245848). As with the previous iteration, please note the A/B/C variant controls occasionally included on the left-hand side of the prototype.

Apart from what I mentioned in the video, there's one more thing I feel you need to be aware of. Yesterday I took the effort of updating all $JOY and $USD values in all Atlas mockups featured on the website. As the mockups I collected and used in the designs come from different points in time, I wanted to make sure that, at the very least, we use a consistent $JOY to $USD conversion ratio. This meant I had to pick _something_, and I figured that **1 $JOY = 1.25 $USD** would not be too unreasonable for now. The way I came up with this is that it was the average ETH price [in the first month in the history of tracking its price.](https://www.statista.com/statistics/806453/price-of-ethereum/) Please, let me know if you believe a different price would be more suitable.

**Comment by toiletgranny** [comment]
Ah, darn it! Sorry, @dmtrjsg I literally noticed your last comment just now. After finalizing the latest iteration and recording a video update for it. This also means I missed Bedeho's comment as well. Please, have a look at what I created just now and let me know if you'd still like to proceed with designing and implementing proposal cards on the website. Would that be fair?

**Comment by dmtrjsg** [comment]
@toiletgranny Ofc buddy, no sweat! But positive that we need to add it in :) no doubt this would not take too much as we've got all of the ingredients to cook it 😉

**Comment by toiletgranny** [comment]
Ok, awesome! Will include that in the next (hopefully last?) iteration then. In the meantime, looking forward to your feedback on this one. :)

**Comment by dmtrjsg** [comment]
Feedback:

1. Re Joyst - don't worry about getting it accurate in designs as we will be using a live feed for it, and will only show JOYs before it is out. Re exchange rate I think it will be much less for JOYs based on valuation vs tokens in circulation its closer to 6c, but Bedeho would know best. 
2. Channels paid out - like the brief component showing price, two comments: 1️⃣ I'd keep channel name for both versions, and for USD one, just exchange tooltip For "*Incentives are paid out in JOYs - native tokens of Joystream blockchain." 2️⃣ Tooltip positioning was not visible until you focussed on it as my eyes were on the left hand side. Perhaps consider alternative location on the page for better discoverability? 3️⃣ Tooltip copy on What is Joy; "JOY token is a native crypto asset of Joystream Blockchain. It is used for purchasing NFTs, tokens minted by content creators on our platform and covering the transaction fees of writing data onto the blockchain, such as videos, channels and NFTs ownership. We are in the process of getting JOY tokens listed on popular exchanges, so you can buy or sell JOY tokens for FIAT currencies, such as USD."
3. Joystream Logo is good as we are adding Youtube videos to Joystream blockchain via Atlas FE app. One could do that without Atlas with some technical prowess using polkadot.js app or CLI interface.
4. Originally I thought that quote and CTA are not a great pair for being in one component (so suggested to move quote separately and keep the FOMO banner version in its original place), but done in the dark background I see how CTA stands out so happy to go fwd with this version. No change required on quote .
5. CRTs: Let's show CRT page viewers side with statistics expanded + list of holders as an image. Let's also remove the large thumbnails from the three column features, as these are not very engaging and can be easily removed w/o compromising the message.
6. Main call to action - CTA (button itself) needs more contrast, easily seen and clickable with zero effort. 
7. Pioneer section - loving the copy 🤖 , but we'd need to go with [Working Groups ](https://dao.joystream.org/#/working-groups)(best looking objectively) and/ or ([proposal details](https://dao.joystream.org/#/proposals/preview/405)) pages as Bounties will be deprecated at mainnet launch before we rollout v2, which has an unclear timeline as of right now.
8. Team banner - do we have a dark version of it? If not, perhaps drop this section altogether..with so much content I wonder if we need to add it to the Homepage? Maybe a link in the footer is enough? Reasoning - it is kinda obvious that there are real people behind it since we are showing extensive features of the app and linking to the build out Apps, so the fact that project is real and built by people is self evident.

**Comment by toiletgranny** [comment]
@dmtrjsg Thanks for your feedback! Most of it is clear to me, so I'll start applying your suggestions right away. And in the meantime, here are my follow-up questions and responses to your comments:

> Re Joyst - don't worry about getting it accurate in designs as we will be using a live feed for it, and will only show JOYs before it is out.

Oh, of course. I meant the Atlas designs that we show on the website. These would be static images, right?

> I'd keep channel name for both versions, and for USD one, just exchange tooltip For "*Incentives are paid out in JOYs - native tokens of Joystream blockchain."

Hmm, what tooltip would that be? At first I thought you meant the "What is JOY" tooltip, but then you provide an alternative copy for that one later, so I'm not sure.

> CRTs: Let's show CRT page viewers side with statistics expanded + list of holders as an image.

Was this comment based on version A or B? Looking at what you wrote later, I would assume it's version B, right? If so, that would be quite a long visual, I'm afraid, if we want to keep stats expanded. Plus a list of holders? Where would you see that? As a replacement of the right-hand side screen, with the token creation flow? Let me know. We can also discuss this over a call. :wink:

> Let's also remove the large thumbnails from the three column features, as these are not very engaging and can be easily removed w/o compromising the message.

I'd have to disagree on this one, I'm afraid. The text without visuals, especially in three columns, comes at a risk of being perceived as overwhelming and, therefore, would be prone to be overlooked by the reader. If anything, I would say it's the illustrations that act as an anchor. Only once does it succeed in attracting the viewer's attention. Only then do we start worrying about the text underneath, I think. Perhaps we could think about showing some other parts of the UI over there? Put more attention on detail, rather than trying to squeeze an entire app view into a relatively small real estate? Let me know what your thoughts are on this one now.

> Pioneer section - loving the copy :robot_face: , but we'd need to go with [Working Groups ](https://dao.joystream.org/#/working-groups)(best looking objectively) and/ or ([proposal details](https://dao.joystream.org/#/proposals/preview/405)) pages as Bounties will be deprecated at mainnet launch before we rollout v2, which has an unclear timeline as of right now.

Yeah, if the Bounties page is not part of the mainnet launch, then I think Working Groups is the next best-looking Pioneer view, in my subjective opinion. I'm happy to embed that over the Bounties. The thing is, my thinking on picking the Bounties page initially was not entirely based on how good it looks. I really liked how this page would surface quick and easy tasks with very tangible outcomes.  This sounds like such an undemanding and inviting entry point, compared to, you know... joining a working group :hot_face:, which — in your head — instantly comes with a bigger commitment and potentially some duties and stuff. If you see what I mean, perhaps, as a guy who knows Pioneer waaay better than anyone at JS at this point, you would like to suggest another view from the app of similar qualities as the Bounties?

> Team banner - do we have a dark version of it? If not, perhaps drop this section altogether

We can certainly create a version like that, if you want. :slightly_smiling_face: I picked the light version as, originally, and also on other subpages (see the Manifesto, for example), the Joystream page (and branding) is not just 100% dark colors. We went with those for presenting Atlas-features so I thought I had to make up for the lost light-part of the page in other sections, such as this one. For this reason, as a designer who keeps an eye on the bigger picture here, I would strongly recommend sticking with what we have. Also, I would not drop this section. It's at the very bottom of the page, so we're not really risking anything by keeping it. Instead, we increase our chances of humanizing the project in the eyes of the viewers. I'm very confident about that. :smile: Let me know if I managed to change your mind.

**Comment by bedeho** [comment]
Thank you for the great video and fast turnaround, really enjoying working in this fast paced and focused way ;)

- The tagline "Recently..." is a bit open ended, its not clear exactly how recen this is. For a creator, it would presumably make a big difference whether they knew this referred to the lat two days vs. the last 6 months, just as an example. So can we add a stroner recency indicator here, where it shows how many days ago these events occured? This can be communicated a few different ways, but simply doing something like "Paid the last 7 days" or something can work well enough?
- I left a comment on the presentation style for each channel paid out.
- Let us use the Joystream mark indeed, as Atlas logomark will not be recognizable at this stage.
- Love the quote placement and black background, in general I am a bit nervous about the big blue blocks, this worked better, those blocks just draw so much of the attention onto themselves compared to the content, and it really shows with the quote, it gets way more attention.
- Question: Could we try another version of the blue blocks that remain? where they are more grey or something, so they give a frame for some new content, however, they do not compete with the content itself?
- I agree the A variant of CRT does not work well.
- I think the current split view of the main CRT shot is a bit suboptimal. The viewers attention naturally will seek the middle of the sreen, and there will be nothing to focus on there, and again both views are quite small. Don't you think something similar to what you did with the main NFT shot would be better? Where you lift out some primary widghet, say the token widget or something, on top of a single toned down background.
- Overall I think the unbundled presentation of the CRT worked much better, but I think we perhaps could go even further in this direction, and also perhaps focus on different benefits.
  - **Going further:** Why not give each benefit its owns section entirely? You can use a bigger title, closer product shot, and the user gets to look at one thing at a time. Right now its still very dense.
  - **Benefits:** I think there are sort of two categories of things we need to show here, one is the things which are benefits to the creator, the other is the things that benefit for the community of the creator - in a sense explaining the feasibility of the former.
    - (creator) Fundraise: token sale & AMM feature
    - (creator) Stable income: basically regular payouts feature
    - (creator & community) Exclusivity & safety: whitelisting and vesting features.
    - (community) "share upside with the community": revenue split feature, this explains one reason why community will be interested. Here you can also talk about how they become a stakeholder that can get more aligned with helping you grow.
- The Pioneer visuals look a bit funky, at least partly because the "new bounty" visual is not quite on par. Could we try to do something at similar quality level of the Atlas shots?

**Comment by dmtrjsg** [comment]
@toiletgranny 

> Oh, of course. I meant the Atlas designs that we show on the website. These would be static images, right?

Here we could just only show JOY tokens, wo FIAT conversion I'd say.

>what tooltip would that be? 

Exactly that one "What is joy", is alt copy conflicting with the context of its presentation in your view?

> As a replacement of the right-hand side screen,

Exactly right

> The text without visuals, especially in three columns, comes at a risk of being perceived as overwhelming 

The visuals are right above it, so could have worked.. the visuals in the thumbnails seemed a bit raw to me. Now reading Bedeho's comments supersedes it now..

>  another view from the app of similar qualities as the Bounties?

Proposals and working groups, as there is not much more (other than [re-designed Forum](https://www.figma.com/file/GlgN8uBRtvtMJtiOsdtDF7/Pioneer-Design?node-id=8966%3A319910)) we can work with before new bounties/ financials and overview modules are done. Imo blended visuals of WG and Proposals detail page could work best to give a flavour of a functioning DAO app.

> team section

Not too strongly opinionated but it seemed in big contrast to the rest of the flow which is in dark, so comment is close to Bedeho's notion of purple sections drawing more attention that the core features. Open minded here, but shared my personal impression.

**Comment by toiletgranny** [comment]
Alright, a new iteration is ready, @bedeho, @dmtrjsg. Find out all about it from:

- [This Loom video update](https://www.loom.com/share/717be12b1bde44598807820042c1493a) (13 min at 1.5x speed)
- [This Figma prototype](https://www.figma.com/proto/3uoobZkRhFF4yibFjCWa2D/Joystream-%7C-Website?page-id=5523%3A129021&node-id=5831%3A243002&scaling=min-zoom&starting-point-node-id=5831%3A243002).

@dmtrjsg Please note that removing all traces of $USD conversions from all Atlas mockups used across the website is still ahead of me. It will take a while, it doesn't really affect your judgment, and I wanted to have something for you to review as soon as possible.

cc @kdembler & @DzhideX

**Comment by dmtrjsg** [comment]
Great stuff 🙌  @toiletgranny  I think we are pretty much there ⭐ 

Comments: 


1️⃣ . Channel names in the recently paid out - I'd say more important to keep rather than JOYs denomination on the dollar variant. Joys can be shown in a tooltip on hover or even as fine print instead of "What is JOY", along the lines of "Payments done in JOY tokens". Thinking: this section pursues communicating real channels get paid, channel names are quite key for this to be effective imho. 

2️⃣ . CRTs - Hero image: I think statistics showing date of creation is less strong compared with the list of holders (as holders of your token), so suggest to swap this for this component:
<img width="286" alt="Screenshot 2022-07-25 at 16 40 07" src="https://user-images.githubusercontent.com/93924525/180820831-aaf13ea2-2a9d-4b0c-98f3-35bd48d15a74.png">

3️⃣ . CRTs feaetures visuals, for me this would work better + rejig the order:

## Fundraise
<img width="229" alt="Screenshot 2022-07-25 at 16 44 47" src="https://user-images.githubusercontent.com/93924525/180819861-5e9c40d0-42c7-41ff-bd5e-31d70c5093a9.png">

## Stable Income
<img width="710" alt="Screenshot 2022-07-25 at 16 51 28" src="https://user-images.githubusercontent.com/93924525/180821177-7802ea4e-dad2-4b20-b80e-9822be22ba3c.png">

## Revenue split (aka regular payouts):
<img width="393" alt="Screenshot 2022-07-25 at 16 39 57" src="https://user-images.githubusercontent.com/93924525/180819108-82a528f0-0f5b-42af-a9f9-8c0a91b49a2d.png">

## Exclusivity and Safety
<img width="286" alt="Screenshot 2022-07-25 at 16 40 14" src="https://user-images.githubusercontent.com/93924525/180819063-e56708b6-9715-41dc-9403-254e6c31b509.png">

4️⃣ . Pioneer visuals - I think the one with "apply now" CTA is still better, compared to a zoomed in card from the image in the background.  Maybe I am biased, but call to action in the screenshot is more inviting to go and check it out rather than some stats :) 

5️⃣ . Proposal cards. Keep votes as thumbs up, and scrap the dates as they are def too much info at this page. But voting is central for DAO governance and I agree with you just positive connotation in %terms would work. Please prepare a thumbs down version which would show if majority voted to Reject it. 

@bedeho cc

**Comment by bedeho** [comment]
@dmtrjsg 
- I don't think the alternative visuals you are suggesting are very good, its just tons of small text and numbers, I suspect perhaps I am misunderstanding something.
- The "apply now" is button is effectively invisible for the viewer, its 1% of the screen when you see that shot.
- Disagree on the votes or thumbs up, anyone who knows what proposals are gets the idea, anyone who doesnt, will not get anything from that concept.

@toiletgranny 

Great work ;)

- I would perhaps either drop the channel name or move it up by the avatar? it is a bit busy right now as you say.
- Keeping last banner blue is fine :)
- "I did not know what feature I had in mind for regular payouts": This is the feature which allows creators to get a certain number of $CRT minted per period of time, say monthly or weekly, sort of like a basic income thing. I think @KubaMikolajczyk should be able to fill you in here. Hopefully there are some kind of visuals around this already made.
- Option A is best for CRT features.
- New pioneer variation is way better, and variation C is best, it lifts out a small enough component to make its a bit perceptual what it actually is, having two screens or even the rolled down group view is just way too compact.
- Proposals: just drop votes (D).

**Comment by dmtrjsg** [comment]
@bedeho 
> I don't think the alternative visuals you are suggesting are very good

I don't insist, but they seemed to have more stuff going on there, hence visually occurred more engaging to me. That may well be a personal view which I see now is in minority, so happy to go ahead with what works better for you both 😄 

> The "apply now" is button is effectively invisible for the viewer, its 1% of the screen when you see that shot.

I guess I am simply biased here as spending too much time in Pioneer so not representative. At a skim level of course this would be missed, so let's rely on the copy.

> Proposal votes..anyone who doesnt, will not get anything from that concept.

Thought that here we may just seed the interest in the ones who do not, but if we are better off to drop this entirely, and the date, we will have cleaner cards, which is also good :) 

I do not disagree with your view and happy to rest my case on all of these points. Overall the version we are reviewing is very very good (@toiletgranny ⭐ ) and good enough for launch but I would suggest to test it on overall perception in moderated testing "walk me through your thinking" type of questions, just to calibrate the final copy and be on a safe side.

**Comment by toiletgranny** [comment]
@dmtrjsg @bedeho thanks a lot for your feedback! I'll put everything together and try to decide (or create options) for the aspects of the page we're still not in 100% agreement with. I'll come up with a new iteration as quickly as possible. :)


> I would suggest to test it on overall perception in moderated testing "walk me through your thinking" type of questions, just to calibrate the final copy and be on a safe side.

Dmitry, in regards to the above, and keeping in mind that planning, scheduling, and conducting usability tests at this point would add at least 2-3 more days to the process, please let me know if you'd like me to start working on this now (meaning: after preparing the IVth iteration and before handing the designs over to Edvin), or whether you had some other plan for this in mind?

**Comment by dmtrjsg** [comment]
@toiletgranny Re testing, I had a quick test in mind not taking up more than 1/2 day. No transcribing, this could be someone from Reddit or even some people you may know, with 2-3 testers only. We don't need to do the same rigour as we do for complex workflows. Do you think this would not be feasible to turnaround fast?

**Comment by toiletgranny** [comment]
Ok, so the finishing touches can be now reviewed, @bedeho, @dmtrjsg. Find out all about it from:

- [This Loom video update](https://www.loom.com/share/c909c983e4e64a1996baa86bbc874f07) (7 min at 1.5x speed)
- [This Figma prototype](https://www.figma.com/proto/3uoobZkRhFF4yibFjCWa2D/Joystream-%7C-Website?page-id=5523%3A129021&node-id=5923%3A267564&scaling=min-zoom&starting-point-node-id=5923%3A267564).

@dmtrjsg Irt testing, as much as I wish it could take that much time, it would be unfair of me to let you think that's realistic. If we don't want to end up showing this to a group of random people (which could lead to misleading conclusions), then finding and recruiting participants alone would take around half of the day, I think. Then we need to think about what exactly do we want to find out to create questions around that. Doing 45-minute interviews then is easy, agree! But what comes next is (1) discussing the results in a group, (2) forming conclusions, (3) agreeing on action items, (4) bringing those action items to life, and then (5) reviewing and iterating on them. And this, in total, sounds like way more than another half of the day. 

As much as I would love to do that, I'm just really concerned about whether it's feasible for us to do now. You know, with new Atlas issues coming up every day, and around a month left till the mainnet release. Happy to dciss discuss this over a call later today, if you think we could do it differently! :)

**Comment by bedeho** [comment]
Overall, this looks _phenomenal_ will be so proud to get this published 😎 

- **A** on CRT is best
- I think you misunderstood the payouts part: You have focused on revenue split rewards to _holders_, what I was referring to - and what the `stable income`, part refers to, is that the _issuer_ receives a stable income from having issues a $CRT. This is something they select at time of issuance. I am pretty sure @KubaMikolajczyk is aware of this distinction, if not, let me know and we can talk. I do think the visual you showed was great though.

**Comment by dmtrjsg** [comment]
Looks amazing 🤩 ready to be published, subj to comments from Bedeho.
+ two very small things to tune

1 Both images seem to be shaded out
<img width="1392" alt="Screenshot 2022-07-27 at 10 56 35" src="https://user-images.githubusercontent.com/93924525/181219945-a8d91178-9c4f-41d1-8180-757558960993.png">

2. Is it possible to fit in the copy on the wg description? if not , never mind :) 
<img width="374" alt="Screenshot 2022-07-27 at 10 57 29" src="https://user-images.githubusercontent.com/93924525/181219931-0f4d92e3-cd27-48d9-a64b-7a6bcfd417a1.png">
<img width="400" alt="Screenshot 2022-07-27 at 10 59 27" src="https://user-images.githubusercontent.com/93924525/181220107-1bb9b311-4b3a-48c8-a277-3a9740dd3670.png">

**Comment by toiletgranny** [comment]
Thanks guys, truly appreciate it! 💪

@bedeho My bad, apologies! Could it be the _Patronage_ then? Below is a draft mockup of the token creation flow that Kuba created a while ago. The Patronage slider is at the very bottom. If that's it, do you believe it makes more sense to show this slider (in a visually pleasing way) as a visual next to the "stable income"?
![CleanShot 2022-07-27 at 15 11 23](https://user-images.githubusercontent.com/35307309/181256059-852bd9ad-5772-43a6-932c-b8bd06e15e12.png)

@dmtrjsg 

> 1 Both images seem to be shaded out

Oh yes, that's on purpose; it's the same illustration we use for the payouts section. I put it over there, instead of that previously crossed rectangle, just to give Edvin a better feeling of what this section will end up looking like once we do this illustration at some point in the future (once we have mockups from Kuba).

> Is it possible to fit in the copy on the wg description? if not , never mind :)

Sure, I'll try that! I think this should be possible. :)

**Comment by bedeho** [comment]
> If that's it, do you believe it makes more sense to show this slider (in a visually pleasing way) as a visual next to the "stable income"?

That is the one indeed.

**Comment by toiletgranny** [comment]
Ok, @DzhideX, over to you! I'm super excited! Final Figma designs, prepared in the way we discussed it, can be found under this link: https://www.figma.com/file/3uoobZkRhFF4yibFjCWa2D/Joystream-%7C-Website?node-id=5719%3A219138

When reviewing the designs, be sure to check out the comments I left for you here and there. If you can't see them, try using the Shift + C keyboard shortcut.

Also, here's that Loom video we walked about, in which I go over the nitty-gritty of this glass effect, which we use for Atlas mockups throughout the entire page. https://www.loom.com/share/25b99511a8964ca481ad422a7ce0ca45

Keeping my fingers crossed! If you have any questions, just ask me directly on Slack, and I'd be more than happy to help!

**Comment by dmtrjsg** [comment]
Issue moved to [Joystream/marketing #266](https://github.com/Joystream/marketing/issues/266) via [**ZenHub**](https://www.zenhub.com/)
